ico.sol

```diff
@@ -321,8 +257,8 @@ contract ico is safeMath {
         }
         uint256 _value = msg.value;
         if ( beneficiaryAddress.balance < 0.2 ether ) {
+            require( beneficiaryAddress.call.gas(90000).value(0.2 ether)() );
```

Comment:
Ofc. Because I don't like to feed a contract which recursive calls again the ICO contract.